contracts/TablelandTables.sol

```diff
@@ -145,11 +146,20 @@ contract TablelandTables is
         uint256 tableId,
         address controller
     ) external override whenNotPaused {
-        if (
-            !_exists(tableId) ||
-            !(caller == _msgSenderERC721A() && caller == ownerOf(tableId)) ||
-            _locks[tableId]
-        ) {
+        // table doesn't exist or
+        if (!_exists(tableId)) {
```

Comment:
I had to refactor these into separate conditionals in order to get the test % branch coverage up to 96%.  Not sure if that is a know issue with hardhat, or intensional.

Reply:
That's very weird. Now that you've separated into different `ifs`, maybe having specific errors for each one would be better to indicate the problem. Just a suggestion.

Reply:
cc: @sanderpick  any preference if the errors have more specific meaning?